service/experiment-config.yaml

```diff
@@ -2,7 +2,7 @@
 # Unless you are a fuzzbench maintainer running this service, this
 # will not work with your setup.
 
-trials: 20
+trials: 15
```

Comment:
Why change this?

Reply:
I thought 15 was the agreed upon number of trials needed, why do 20. i think 20 was needed when we fail on some trials with these 4-5 days long experiments, now it should 2 days and non-preemptibles will be created after first day.

Reply:
I don't think we agreed 15 was needed. I think we decided 15 is sufficient for non-preemptible experiments since it balances costs and accuracy. We need 20 for statistical power.

Reply:
Done reverted.